modules/persistent-storage-csi-snapshots-delete.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-snapshots.adoc
+
+[id="persistent-storage-csi-snapshots-delete_{context}"]
+= Deleting a volume snapshot
+
+Deletion is triggered based on how you define the VolumeSnapshot object.
+
+.Procedure
+
+To enable deletion of a volume snapshot in a cluster:
+
+. Specify the required deletion policy in VolumeSnapshotClass, as shown in the following example:
+
++
+.volumesnapshot.yaml
+[source,yaml]
+----
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass <1>
+metadata:
+  name: csi-hostpath-snap
+driver: hostpath.csi.k8s.io
+deletionPolicy: Delete <1>
+----
+<1> If `Delete`, the underlying snapshot will be deleted, along with the VolumeSnapshotContent object. If `Retain`, both the underlying snapshot and VolumeSnapshotContent remain.
```

Comment:
One note here - if `Retain` is specified and the VolumeSnapshot is deleted (without deleting the corresponding VolumeSnapshotContent), then the content will remain.